source/renderer/app/themes/daedalus/white.js

```diff
@@ -788,6 +788,9 @@ export const WHITE_THEME_OUTPUT = {
       'rgba(45, 45, 45, 0.5)',
     '--theme-sidebar-category-networkInfo-background-color': '#29b595',
     '--theme-sidebar-category-networkInfo-text-color': '#fff',
+    '--theme-sidebar-sort-button-active-background-color':
+      'rgba(250, 251, 252, 0.1)',
```

Comment:
this color is incorrect (it's light-blue theme color) 
it should be rgba (45, 45, 45, 0.1)